srtcore/core.cpp

```diff
@@ -7807,6 +7807,9 @@ int srt::CUDT::sendCtrlAck(CPacket& ctrlpkt, int size)
     dropToGroupRecvBase();
 #endif
 
+    /* tsbpd thread may also call ackData when skipping packet so protect code */
```

Comment:
```suggestion
    // The TSBPD thread may change the first lost sequence record (TLPKTDROP).
    // To avoid it the m_RcvBufferLock has to be acquired.
```